cli.py

```diff
@@ -0,0 +1,32 @@
+import click
+import mongoengine
+
+from config import Config as config
+from models import mongo, sql
+
+@click.group()
+def cli():
+    pass
+
+@click.command()
+def migrate_stats():
+    for stat in mongo.Statistic.objects:
```

Comment:
Need a way to restart - possibly just clear mongo during upgrade?

Reply:
Scrapped this, I'm going to leave it here for anyone with a local install though. If you come along later, it's easier to run mongoexport, pass the file through | grep -Pav '\x00', and use something like this in python to clean it: 

```import csv
import sys

reader = csv.reader(sys.stdin, delimiter=',')
writer = csv.writer(sys.stdout, delimiter=',')

for row in reader:
    if not all(row):
        print(row, file=sys.stderr)
        continue
    writer.writerow(row)```

Then, run `pv stats.csv | grep -Pav '\x00' | python3 ../fix.py | psql -U stats -c "\copy stats(device_id, model, version, country, carrier, carrier_id, submit_time) from STDIN delimiter ',' csv header;`